CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ All notable changes for each version of this project will be documented in this
 - `IgxCarousel`:
     - **Breaking Changes** -The carousel slides are no longer array, they are changed to QueryList.
     - **Behavioural change** - When slides are more than 5, a label is shown instead of the indicators. The count limit of visible indicators can be changed with the input `maximumIndicatorsCount`
+- Added `Directionality` service providing information about the text direction of the element's content.
```

Comment:
You might wanna drop this from the changelog, if it's not in the public exports, and replace it with added RTL support for the Progress bar